### PR TITLE
Disabled type annotation inspections in IntelliJ.

### PR DESCRIPTION
**Pull request by kshakir**
Synced common scala .gitignore across repositories.

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="TypeAnnotation" enabled="false" level="WARNING" enabled_by_default="false" />
+  </profile>
+</component>
```
Comment (mcovarr) [inline .idea/inspectionProfiles/Project_Default.xml]: \n?

**Comment by mcovarr** [comment]
:+1:

File: .idea/inspectionProfiles/Project_Default.xml (hunk 1 above)
Comment (kshakir) [inline .idea/inspectionProfiles/Project_Default.xml]: Nope. This file is generated by intellij's GUI. Although it wounds me deeply to see github's red not-sign, I'm leaving it there and not hand-editing this file. Though perhaps I'll file a bug against jetbrains. /s

**Comment by ruchim** [comment]
:+1:

**Pull request merged**